### PR TITLE
[AIRFLOW-2513] Change `bql` to `sql` for BigQuery Hooks & Ops

### PR DESCRIPTION
**Pull request by kaxil**
Make sure you have checked _all_ steps below.

### JIRA
- [x] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-2513
    - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a JIRA issue.

### Description
- [x] Here are some details about my PR, including screenshots of any UI changes:
- Change `bql` to `sql` for BigQuery Hooks & Operators for consistency. Discussed in more detail [here](https://issues.apache.org/jira/browse/AIRFLOW-2513)

### Tests
- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
- `test_bql_deprecation_warning`
- `test_nobql_nosql_param_error`
- `BigQueryOperatorTest.test_bql_deprecation_warning`

### Commits
- [x] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### Documentation
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.

### Code Quality
- [x] Passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

**Comment by kaxil** [comment]
PTAL @Fokko @fenglu-g

File: UPDATING.md [hunk 1]
```diff
@@ -63,6 +63,10 @@ Dataflow job labeling is now supported in Dataflow{Java,Python}Operator with a d
 "airflow-version" label, please upgrade your google-cloud-dataflow or apache-beam version
 to 2.2.0 or greater.
 
+### BigQuery Hooks and Operator
+The `bql` parameter passed to `BigQueryOperator` and `BigQueryBaseCursor.run_query` has been deprecated and renamed to`sql` for consistency purposes. Using `bql` will still work (and raise a `DeprecationWarning`), but is no longer
```
Comment (Fokko) [inline UPDATING.md]: missing space in: to`sql`

File: airflow/contrib/hooks/bigquery_hook.py [hunk 2]
```diff
@@ -526,6 +530,23 @@ def run_query(self,
 
         """
 
+        # TODO remove `bql` in Airflow 2.0
+        sql = bql if sql is None else sql
```
Comment (Fokko) [inline airflow/contrib/hooks/bigquery_hook.py]: Maybe add the ticket number to it, for further reference.

File: tests/contrib/hooks/test_bigquery_hook.py [hunk 3]
```diff
@@ -213,9 +214,29 @@ def test_invalid_schema_update_options(self):
                 "test_schema.json",
                 ["test_data.json"],
                 schema_update_options=["THIS IS NOT VALID"]
-                )
+            )
         self.assertIn("THIS IS NOT VALID", str(context.exception))
 
+    @mock.patch.object(hook.BigQueryBaseCursor, 'run_with_configuration')
```
Comment (Fokko) [inline tests/contrib/hooks/test_bigquery_hook.py:220]: Nice one

**Review by Fokko — CHANGES_REQUESTED** [review]
LGTM, minor nits

**Comment by kaxil** [comment]
@Fokko I would like to include this in v1.10 as well. But I didn't merge to v1.10-test branch as I read somewhere in dev list emails that we may rebranch 1.10 from master.

>We will rebranch 1.10 from master.

**Pull request closed without merging**